packages/fields/src/types/OEmbed/views/blocks/o-embed.js

```diff
@@ -21,7 +21,7 @@ const Embed = ({ url, oembedData }) => {
 
   if (options.previewComponent) {
     // The adapter should implement this option
-    const [Preview] = options.adminMeta.readViews([options.previewComponent]);
+    const [Preview] = options.readViews([options.previewComponent]);
```

Comment:
I think this is wrong. The blocks are passed their own copy of `adminMeta` here: https://github.com/keystonejs/keystone/blob/master/packages/field-content/src/views/Controller.js#L57 . You'll need to explicitly pass in `readViews`.

Once #2890 is merged the `adminMeta` copy in `options` can be removed entirely, I think.

Reply:
Tested #2890 👍